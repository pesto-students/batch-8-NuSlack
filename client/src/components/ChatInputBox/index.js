import React from 'react';
import { Icon, Input } from 'antd';
import styled from 'styled-components';
import CircleButton from '../CircleButton';

const InputContainer = styled.div`
  margin-bottom: 0;
`;
const ChatInputBox = () => (
  <InputContainer>
    <Input
      placeholder="Say it"
      size="large"
      prefix={<Icon type="user" style={{ color: 'rgba(0,0,0,.25)' }} />}
      suffix={(
        <>
          <CircleButton>
            <Icon type="smile" />
          </CircleButton>
            &nbsp;
          <CircleButton>
            <Icon type="paper-clip" />
          </CircleButton>
        </>
)}
    />
  </InputContainer>
);

export default ChatInputBox;
