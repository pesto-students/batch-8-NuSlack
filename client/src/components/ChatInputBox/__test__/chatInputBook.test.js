import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import ChatInputBox from '../index';

describe('<ChatInputBox />', () => {
  it('should match the snapshot', () => {
    const tree = shallow(<ChatInputBox />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
