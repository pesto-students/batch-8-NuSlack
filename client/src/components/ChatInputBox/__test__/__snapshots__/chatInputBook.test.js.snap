// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChatInputBox /> should match the snapshot 1`] = `
<styled.div>
  <Input
    disabled={false}
    onChange={[Function]}
    onPressEnter={[Function]}
    placeholder="Say it"
    prefix={
      <Icon
        style={
          Object {
            "color": "rgba(0,0,0,.25)",
          }
        }
        type="user"
      />
    }
    size="large"
    suffix={
      <React.Fragment>
        <index>
          <Icon
            type="smile"
          />
        </index>
        Â 
        <index>
          <Icon
            type="paper-clip"
          />
        </index>
      </React.Fragment>
    }
    type="text"
    value=""
  />
</styled.div>
`;
